docs/quickstarts/subs-getting-started/metadata.yml

```diff
@@ -1,13 +1,13 @@
-kind: QuickStarts
-name: subs-getting-started
-tags:
-- kind: bundle
-  value: subscriptions
-- kind: content
-  value: documentation
-- kind: product-families
-  value: subscriptions-services
-- kind: use-case
-  value: spend-management
-- kind: use-case
-  value: system-configuration
\ No newline at end of file
+ kind: QuickStarts
```

Comment:
@bluemorpho-mpw can we undo these whitespace changes to `subs-getting-started` files? I don't think they were intended.